code/modules/shield_generators/shield_generator.dm

```diff
@@ -38,6 +38,10 @@
 	var/list/mode_list = null			// A list of shield_mode datums.
 	var/emergency_shutdown = FALSE		// Whether the generator is currently recovering from an emergency shutdown
 	var/list/default_modes = list()
+
+	//TODO:Add hacking i guess...
+	var/hacked = 0						// Whether the generator has been hacked by cutting the safety wire.
```

Comment:
I actually intentionally removed hacking and the safety wire when i was porting the shield generator.

All it did was unlock some modes like overcharge, which make perfect sense to have as options anyways. And it's silly to make the technomancers have to hack their own machines every round